### PR TITLE
MuTE Core Integration

### PR DESCRIPTION
**Pull request by BlueWeabo**
After way too many months the first iteration with mutecore has been done. This version has some bugs inside of it, but its fully working otherwise. You all shall decide if you want to actually use it. There are still things to improve as always, but this should be much better than the current state of everything.

Merry Christmass

### CODE REVIEW

File: repositories.gradle [hunk 1]
```diff
@@ -30,4 +30,5 @@ repositories {
             artifact()
         }
     }
+    mavenLocal()
```
Comment (NotAPenguin0) [inline repositories.gradle:33]: Probably unintended

File: dependencies.gradle [hunk 2]
```diff
@@ -53,6 +53,7 @@ dependencies {
     implementation('com.github.GTNewHorizons:Baubles:1.0.4:dev') {transitive=false}
     // Required to prevent an older bauble api from Extra Utilities from loading first in the javac classpath
     compileOnly('com.github.GTNewHorizons:Baubles:1.0.4:dev') {transitive=false}
+    api("com.gtnewhorizons.mutecore:MuTECore:99.99.99:dev")
```
Comment (NotAPenguin0) [inline dependencies.gradle:56]: Probably unintended

**Comment by YannickMG** [comment]
I assume we need a new release of https://github.com/GTNewHorizons/MuTECore before this can meaningfully be reviewed.

**Comment by BlueWeabo** [comment]
Actually, it needs to have a version from this [one](https://github.com/BlueWeabo/MuTECore).  Now if you ask me why it isnt a fork of the GTNH one, I do not quite remember. I can transfer ownership of the one in my profile to the GTNH organisation though, dont need it otherwise.

**Comment by Dream-Master** [comment]
> Actually, it needs to have a version from this [one](https://github.com/BlueWeabo/MuTECore). Now if you ask me why it isnt a fork of the GTNH one, I do not quite remember. I can transfer ownership of the one in my profile to the GTNH organisation though, dont need it otherwise.

merry xmas:)  you can transfer it or i can fork it. Your decission.

**Comment by YannickMG** [comment]
Is the included Coke Oven meant to be a functional example?
image

If I try to get it in item form I get a NPE because `MultiTileBlockRenderer::renderInventoryBlock` passes a null `IBlockAccess` to `MuTERender::render`.

**Comment by BlueWeabo** [comment]
I have added in the checks, but now I am getting a problem with rendering when its placed, and on that point I  am unsure as to why its happening. I can try to figure it out more, but it would have to be after the holidays.

**Comment by boubou19** [comment]
@BlueWeabo Thank you a lot for your hard work, however, MuTEs have become a burden of its own, as the project turned to be too much ambitious and no one wants to pick it up. So we decided to give it up and go back to MTE, which has been in use for more than a decade now. This is done this way so more people can actually give their expertise on the reworked parts of the MTE architecture, with some expected difficulty to do some of the tasks MuTEs wanted to achieve. We also want to get rid of the MuTEs project to yeet all the pressure generated by the community, which are tied to the project.

**Comment by BlueWeabo** [comment]
That is understandable. I wish the best of luck to bring the features wanted from MuTEs into MTE. Thanks for the notice and best wishes to everyone in the dev team!

**Pull request closed without merging**